diff --git a/WORKSPACE b/WORKSPACE
index afd7555..f2e9373 100644
--- a/WORKSPACE
+++ b/WORKSPACE
@@ -631,7 +631,7 @@ new_local_repository(
     # For local MacOS builds, the path should point to an opencv@3 installation.
     # If you edit the path here, you will also need to update the corresponding
     # prefix in "opencv_macos.BUILD".
-    path = "/usr/local",  # e.g. /usr/local/Cellar for HomeBrew
+    path = "/opt/homebrew",  # Apple Silicon Homebrew
 )
 
 new_local_repository(
diff --git a/third_party/opencv_macos.BUILD b/third_party/opencv_macos.BUILD
index 4139f18..7d27b09 100644
--- a/third_party/opencv_macos.BUILD
+++ b/third_party/opencv_macos.BUILD
@@ -35,7 +35,7 @@ exports_files(["LICENSE"])
 # "include/opencv4/opencv2/**/*.h*" and the include prefix needs to be set to
 # "include/opencv4".
 
-PREFIX = "opt/opencv@3"
+PREFIX = "opt/opencv"
 
 cc_library(
     name = "opencv",
@@ -51,8 +51,8 @@ cc_library(
             paths.join(PREFIX, "lib/libopencv_videoio.dylib"),
         ],
     ),
-    hdrs = glob([paths.join(PREFIX, "include/opencv2/**/*.h*")]),
-    includes = [paths.join(PREFIX, "include/")],
+    hdrs = glob([paths.join(PREFIX, "include/opencv4/opencv2/**/*.h*")]),
+    includes = [paths.join(PREFIX, "include/opencv4/")],
     linkstatic = 1,
     visibility = ["//visibility:public"],
 )
