diff --git a/mediapipe/tasks/cc/core/BUILD b/mediapipe/tasks/cc/core/BUILD
index 5bbe625..bf81f75 100644
--- a/mediapipe/tasks/cc/core/BUILD
+++ b/mediapipe/tasks/cc/core/BUILD
@@ -277,7 +277,6 @@ cc_library_with_tflite(
         "//mediapipe/tasks/cc:common",
         "//mediapipe/tasks/cc/core/logging:tasks_dummy_logger",
         "//mediapipe/tasks/cc/core/logging:tasks_logger",
-        "//mediapipe/util/analytics:mediapipe_logging_enums_cc_proto",
         "@com_google_absl//absl/base:core_headers",
         "@com_google_absl//absl/container:flat_hash_map",
         "@com_google_absl//absl/log:absl_log",
diff --git a/mediapipe/tasks/cc/core/logging/BUILD b/mediapipe/tasks/cc/core/logging/BUILD
index 3e44917..224cb58 100644
--- a/mediapipe/tasks/cc/core/logging/BUILD
+++ b/mediapipe/tasks/cc/core/logging/BUILD
@@ -18,19 +18,10 @@ package(default_visibility = [
     "//mediapipe/tasks/cc/core:__subpackages__",
 ])

-cc_library(
-    name = "logging_client",
-    hdrs = ["logging_client.h"],
-    deps = [
-        "//mediapipe/util/analytics:mediapipe_log_extension_cc_proto",
-    ],
-)
-
 cc_library(
     name = "tasks_dummy_logger",
     hdrs = ["tasks_dummy_logger.h"],
     deps = [
-        ":logging_client",
         ":tasks_logger",
         "//mediapipe/framework:timestamp",
         "@com_google_absl//absl/strings:string_view",
diff --git a/mediapipe/tasks/cc/core/logging/tasks_dummy_logger.h b/mediapipe/tasks/cc/core/logging/tasks_dummy_logger.h
index 5960ae4..028601c 100644
--- a/mediapipe/tasks/cc/core/logging/tasks_dummy_logger.h
+++ b/mediapipe/tasks/cc/core/logging/tasks_dummy_logger.h
@@ -22,7 +22,6 @@

 #include "absl/strings/string_view.h"
 #include "mediapipe/framework/timestamp.h"
-#include "mediapipe/tasks/cc/core/logging/logging_client.h"
 #include "mediapipe/tasks/cc/core/logging/tasks_logger.h"

 namespace mediapipe {
diff --git a/mediapipe/tasks/cc/core/task_runner.cc b/mediapipe/tasks/cc/core/task_runner.cc
index d1b6c0b..9b14440 100644
--- a/mediapipe/tasks/cc/core/task_runner.cc
+++ b/mediapipe/tasks/cc/core/task_runner.cc
@@ -42,7 +42,6 @@ limitations under the License.
 #include "mediapipe/tasks/cc/core/logging/tasks_dummy_logger.h"
 #include "mediapipe/tasks/cc/core/logging/tasks_logger.h"
 #include "mediapipe/tasks/cc/core/model_resources_cache.h"
-#include "mediapipe/util/analytics/mediapipe_logging_enums.pb.h"
 #include "tensorflow/lite/core/api/op_resolver.h"

 #if !MEDIAPIPE_DISABLE_GPU
